Add a pgsql container

moolitayer opened the pull request:
postgresql can be accessed using:
```
oc get pods
oc rsh -c postgresql <pod>
PGPASSWORD=redhat123 psql -h localhost
```

jhernand commented on template.yml:
```diff
@@ -185,6 +139,28 @@ objects:
+            protocol: TCP
+          volumeMounts:
+          - mountPath: /var/lib/pgsql/data
+            name: postgresql-94-centos7-volume-1
```
Can we name this just `data`, like we did when it was ETCD?

jhernand commented on template.yml:
```diff
@@ -185,6 +139,28 @@ objects:
           command:
           - /usr/local/bin/clusters-service
+          ports:
+            - containerPort: 8000
```
Try to use the same indentation style that we use in the rest of the file:

```yaml
ports:
- containerPort: 8000
```

jhernand commented on template.yml:
```diff
@@ -185,6 +139,28 @@ objects:
+          ports:
+            - containerPort: 8000
+              name: clusters-svc
+        - name: clusters-db
```
The name of the other container is `service`, not `clusters-service`. Can we name this just `db` instead of `clusters-db`, for consistency?

moolitayer replied:
renaming to postgresql for consistency with the etcd container.

jhernand commented on template.yml:
```diff
@@ -185,6 +139,28 @@ objects:
+            - name: POSTGRESQL_DATABASE
+              value: clusters
+            - name: POSTGRESQL_USER
+              value: postgresql
```
Can we use `service` ad the name of the user?

moolitayer commented:
Addressed comments, see [this](https://github.com/container-mgmt/dedicated-portal/pull/44#issue-197113295) for accessing postgres in a cluster.

cc @nimrodshn

nimrodshn commented:
@moolitayer  nice !

jhernand commented:
@nimrodshn Please remember to do the same for the customers service.
